package.json

```diff
@@ -3,22 +3,19 @@
   "version": "0.5.0",
   "description": "React Hooks library for remote data fetching",
   "main": "./dist/index.js",
-  "module": "./esm/index.js",
-  "react-native": "./esm/index.js",
-  "types": "./dist/index.d.ts",
+  "module": "./dist/index.esm.js",
+  "react-native": "./dist/index.esm.js",
+  "types": "./dist/types/index.d.ts",
   "files": [
-    "dist/**",
-    "esm/**"
+    "dist/**"
   ],
   "repository": "vercel/swr",
   "homepage": "https://swr.vercel.app",
   "license": "MIT",
   "scripts": {
-    "build": "npm run build:esm && npm run build:cjs",
-    "build:cjs": "ncc build src/index.ts -o dist -m -e react",
-    "build:esm": "tsc --module ES6 --outDir esm",
-    "watch": "tsc --watch",
-    "watch:esm": "tsc --module ES6 --outDir esm --target esnext --watch",
+    "build": "bunchee src/index.ts --no-sourcemap -m",
```

Comment:
Unsure thing about whether to keep sourcemap here. Might increase the debug difficulty on production with minified code only for users.

Reply:
I think for now let's include it as it helps identifying the original code, but I doubt if anyone is using it.